pyproject.toml

```diff
@@ -28,6 +28,7 @@ click = "^8.1.7"
 python-dotenv = "1.0.0"
 embedchain = "^0.1.98"
 appdirs = "^1.4.4"
+agentops = { version = "^0.1.1", optional = true }
```

Comment:
Updating it to be an option dependency, so we might need to add some fallback when importing the agent ops